libs/metrics-wai/src/Data/Metrics/Middleware/Prometheus.hs

```diff
@@ -33,12 +34,17 @@ import Network.Wai.Routing.Route (Routes, prepare)
 -- This middleware requires your servers 'Routes' because it does some normalization
 -- (e.g. removing params from calls)
 waiPrometheusMiddleware :: (Monad m) => Routes a m b -> Wai.Middleware
-waiPrometheusMiddleware routes =
+waiPrometheusMiddleware routes = waiPrometheusMiddlewarePaths $ treeToPaths $ prepare routes
```

Comment:
I'm a bit surprised by our lack of test coverage for `proxy` :roll_eyes: 

Wouldn't this ticket have been a chance to make the beast testable and ensure we're not breaking anything? :thinking: (I may miss some details something, though.)

Reply:
i would argue "out of scope", this is only about internal end-points, and i only changed Run.hs because it seemed the most straight-forward way to do this at the time. it's tricky to test these without api keys for the resp. services. and the public routes really haven't changed in this PR, just wrapped a little.

but i agree to your point.  hm, what to do?